### DISCUSSION TITLE
how to use github.com/torquem-ch/mdbx-go for golang?

### DISCUSSION THREAD

**Discussion by hiqsociety** [erthink/libmdbx#351] [Q&A] [answered]
Does anyone know how to implement this? There's no basic tutorial on how to get it working.
my golang uses boltdb and i'm trying to convert it to use libmdbx

**Comment by erthink** [comment, marked as answer]
Take look to https://github.com/erigontech/erigon/tree/main/db and @AskAlexSharov.
